project/plugins.sbt

```diff
@@ -1,2 +1,3 @@
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
```

Comment:
publish for publishing snapshots, release plugin for executing a release procedure (which includes publishing a non-snapshot)?